### PR TITLE
Allow admin to disable MFA for a user

### PR DESCRIPTION
**Pull request by knolleary**
## Description

Allows an admin user to disable MFA on a users account.

In absence of providing users recovery codes, this will be one path for a user to get back into their account via a support request.

### CODE REVIEW

File: frontend/src/pages/admin/Users/General.vue
Comment (Pezmc) [inline frontend/src/pages/admin/Users/General.vue:61]: Where is `sso_enabled` coming from?

**Review by Pezmc — APPROVED** [review]

File: frontend/src/pages/admin/Users/General.vue
Comment (knolleary) [reply frontend/src/pages/admin/Users/General.vue:61]: It's a pre-existing flag on the user object. Thought it useful to add to what we expose here... although it's getting to a point where this table needs a bit of redesign.

**Comment by knolleary** [comment]
This can be merged once #3065 has been reviewed. I'm holding off merging it now to avoid complicating the 3065 review.

**Pull request merged**
